### PR TITLE
Redux setup

### PR DESCRIPTION
**Pull request by eddielo91**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
```
Comment (nick567187) [inline .circleci/config.yml:5]: cool idea for versions

File: README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@
 
 An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 
-- Node 6.13.0
-- etc
+- Node 8.11.3
```
Comment (nick567187) [inline README.md:26]: This Node version is too new

File: client/action/fetch.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+});
+
+const loadingData = (bool) => ({
+  type: 'LOADING_DATA',
+  isLoading: bool
+});
+
+const fetchSuccess = (data) => ({
+  type: 'FETCH_SUCCESS',
+  data
```
Comment (nick567187) [inline client/action/fetch.js:13]: lone data key? what is it?

File: client/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const { combineReducers } = Redux;
+const { fetchErrorReducer, loadingDataReducer, fetchSuccessReducer } = require('./fetch.js');
+
+const rootRuducer = combineReducers({
```
Comment (nick567187) [inline client/reducers/index.js:4]: Reducer

**Review by nick567187 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eddielo91) [reply README.md:26]: But Why?!

**Pull request merged**